components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go

```diff
@@ -142,13 +142,15 @@ func getRequestsForPostgresAsSource(config *models.Config) []models.FirewallRequ
 				continue
 			}
 			//for Upd
-			reqBody := models.FirewallRequest{
-				SourceNodeIP:               sourceNodeIP,
-				DestinationNodeIP:          destNodeIP,
-				DestinationServicePort:     "9638",
-				DestinationServiceProtocol: "udp",
+			if config.DeploymentState != "post-deploy" {
```

Comment:
make this "post-deploy" a const, then compare it here.
```suggestion
			if config.DeploymentState != POST_DEPLOY_STATE {
```

Reply:
done